control/tests/iosys_test.py

```diff
@@ -802,10 +802,64 @@ def test_named_signals(self):
         )
         ss_feedback = ct.feedback(self.mimo_linsys1, self.mimo_linsys2)
         lin_feedback = ct.linearize(ios_connect, 0, 0)
-        np.testing.assert_array_almost_equal(ss_feedback.A, lin_feedback.A)
-        np.testing.assert_array_almost_equal(ss_feedback.B, lin_feedback.B)
-        np.testing.assert_array_almost_equal(ss_feedback.C, lin_feedback.C)
-        np.testing.assert_array_almost_equal(ss_feedback.D, lin_feedback.D)
+        for M, N in ((ss_feedback.A, lin_feedback.A), (ss_feedback.B, lin_feedback.B),
+                     (ss_feedback.C, lin_feedback.C), (ss_feedback.D, lin_feedback.D)):
+            np.testing.assert_array_almost_equal(M, N)
```

Comment:
Please avoid loops like this. It makes debugging failing tests in pytest more difficult, because the assertion error message now reads "M is not equal N" instead of directly showing which one failed.

Reply:
Yes, totally. I'll revert back in the next commit. The three previous checks in this test were using this "for M, N" structure, so I figured this one was an anomaly and changed it. I will remove all for loops then.